### PR TITLE
Feature/296 allow editing site title

### PR DESCRIPTION
**Pull request by NotExpectedYet**
Add's in an environment variable option to allow editing site title. 

Did not feel this needs adding automatically to the environment file. 

I think I hooked in right, just let me know if I missed anything

### CODE REVIEW

File: server_src/routes/index.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ async function welcome() {
       } else {
         res.render("welcome", {
           page: "Welcome",
+          octoFarmPageTitle: process.env.OCTOFARM_SITE_TITLE,
```
Comment (davidzwa) [inline server_src/routes/index.js:46]: Try to use the process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY] so we decouple the name from the variable intended

Comment (NotExpectedYet) [reply server_src/routes/index.js:46]: Will PR now, aye was a bit but it works and it's in one place.

File: app-env.js [hunk 2]
```diff
@@ -234,6 +234,18 @@ function getViewsPath() {
   return viewsPath;
 }
 
+function ensurePageTitle() {
+  if (!process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY]) {
+    logger.info(
+        `~ ${AppConstants.OCTOFARM_SITE_TITLE_KEY} environment variable is not set. Assuming default: ${AppConstants.OCTOFARM_SITE_TITLE_KEY}=${AppConstants.defaultOctoFarmPageTitle}.`
+    );
+    printInstructionsURL();
```
Comment (davidzwa) [inline app-env.js:242]: I would not log this message and would avoid printing instructions. I think this needs to be a silent setting just like prerelease (which is gonna be removed, but still).

Comment (NotExpectedYet) [reply app-env.js:242]: I'm sure I removed that but will do another push for it aha

Comment (davidzwa) [reply app-env.js:242]: Ill do some pushups for too rapid rebasing!

Comment (NotExpectedYet) [reply app-env.js]: bahaha, I've lifted enough tables for the day does that count? 

Fixed anyway

**Review by davidzwa — CHANGES_REQUESTED** [review]
image

**Review by davidzwa — APPROVED** [review]

**Comment by NotExpectedYet** [comment]
> image

That's because off my merge conflict. The defaults always set. I'll fix.

**Review by davidzwa — APPROVED** [review]

**Pull request merged**